test/builtin_dataset_mocks.py

```diff
@@ -370,8 +370,8 @@ def cifar100(info, root, config):
     return len(train_files if config.split == "train" else test_files)
 
 
-# @register_mock
-def caltech101(info, root, config):
+@register_mock(configs=[dict()])
```

Comment:
We could also make this the default value, but I think being explicit here is fine,